main/test/flix/Test.Exp.Interpolation.flix

```diff
@@ -111,6 +111,101 @@ namespace Test/Exp/Interpolation {
         let x = "a";
         "$${x}$" == "$a$"
 
+    @test
+    def interpolation21(): Bool =
+        let x = true;
+        "${x}" == "true"
```

Comment:
General question. Scala and Python (and probably others) have a prefix to mark that a string is interpolated, which lets a programmer explicitly indicate their intention to use interpolation. What influenced the choice not to use a prefix?

Reply:
Good question: I have no answer. I was thinking about this yesterday.

it seems to me that Python/Scala etc. all added interpolation later hence they were forced to come up with a new type of string. But for us, why not have all strings be interpolation? I see very little downside. If you want to write the string "${x}$ you can also just do "$" + "{x}". (or we could add escapes).

So, should all strings be interpolated?

Reply:
I think that's fair enough. This will be even less of an issue when we get raw strings.